test/searchRestaurant.test.ts

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+import { search } from '../src/API';
+jest.mock('axios');
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+
+describe('Models -> searchRestaurant', () => {
```

Comment:
Apprezzo molto il testing, e nel frontend è una cosa sicuramente da migliorare e che ci manca, però in questo caso... cosa stiamo testando esattamente? perché mi sembra quasi una tautologia, forse non stiamo testando il nostro codice, che fa solo una get, ma stiamo solo testando axios... quindi in questo caso non ci vedo valore, lo toglierei per reinserirlo quando avremo un pò di logica vera da testare